src/utilities/FormManager.ts

```diff
@@ -286,8 +286,10 @@ class FormManager {
     }
   }
 
-  public async onSave(event: any) {
-    event.preventDefault();
+  public async onSave(event?: any) {
+    if (!!event) {
```

Comment:
Hm, technically, the `!!` doesn't affect the if statement, but I'm also fine with it for readability, your call.

```suggestion
    if (event) {
```

Reply:
no! ill take that suggestion. it's simplier